test-network-nano-bash/bft-config/configtx.yaml

```diff
@@ -287,28 +287,28 @@ Profiles:
           Host: 127.0.0.1
           Port: 6050
           MSPID: OrdererMSP
-          Identity: ../crypto-config/ordererOrganizations/example.com/orderers/orderer.example.com/msp/signcerts/orderer.example.com-cert.pem
+          Identity: ../crypto-config/ordererOrganizations/example.com/orderers/orderer.example.com/msp/signcerts/cert.pem
```

Comment:
I think this will break the default scenario where cryptogen is used.
Maybe when the certs are created using CA in createEnrollments.sh we need to copy the pem file to the existing name produced by cryptogen.